src/development/ui/advanced/typography.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Flutter's fonts and typography
+description: Learn about Flutter's support for typography.
+---
+
+If you aren't familiar with [_typography_][], it essentially
+means how text is styled. [Variable fonts][] (also
+called OpenType fonts), allow you to control various
+aspects of text styling, such as font weight, width, slant,
+and so on.
+
+## API docs
+
+Here are some Flutter classes that allow you to
+modify variable fonts:
+
+* [`FontFeature`][] allows you to select glyphs in a font
+* [`FontVariation`][] allows you to modify a font's design axes
+
+## More info
+
+If you would like to read about one engineer's experience
+customizing variable fonts and animating them as they
+morph, check out [Playful typography with Flutter][],
+a free article on Medium. The associated example also
+uses a custom shader.
```

Comment:
**issue:** This reads with difficulty. Consider breaking it down into more sentences.
```suggestion
To read about one experience, check out [Playful typography with Flutter][].
This Medium article details how one engineer customized variable fonts
and animated them as they morph. The associated example also used a
custom shader.
```

This reduced readability from Grade 14 to Grade 7 with the same number of words.